packages/config/config/devices/0x008b/trane_xr524.json

```diff
@@ -257,47 +302,57 @@
 		},
 		{
 			"#": "75",
-			"label": "Manually Run Fan",
-			"unit": "minutes",
+			"label": "Recovery Enable",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 120,
-			"defaultValue": 0
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "76",
-			"label": "Run/Hold Schedule",
+			"label": "Schedule",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 1,
+			"defaultValue": 0,
 			"allowManualEntry": false,
 			"options": [
 				{
 					"label": "Hold",
 					"value": 0
 				},
 				{
-					"label": "Run schedule",
+					"label": "Run Schedule",
 					"value": 1
 				}
 			]
 		},
 		{
 			"#": "77",
-			"label": "2nd Stage Heat Enable",
+			"label": "ESM",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 1,
+			"maxValue": 2,
 			"defaultValue": 0,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "no 2nd stage heat",
+					"label": "OFF",
 					"value": 0
 				},
 				{
-					"label": "2nd stage heat",
+					"label": "ON",
```

Comment:
Disable/Enable please

Reply:
OFF/ON is matching XR524 UI and user manual.
Should it match UI/manual or be enable/disable?

Reply:
see previous comment.